service/src/main/java/uk/gov/hmcts/reform/fpl/request/AuthorizationConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.request;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.context.annotation.RequestScope;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Configuration
+public class AuthorizationConfiguration {
+
+    @Bean
+    @RequestScope
+    public RequestData authorisation(HttpServletRequest request) {
+        String authorization = request.getHeader("authorization");
+
+        return new RequestData(authorization);
+    }
+}
```

Comment:
request data does not have to be request scoped, by default injected HttpServletRequest is request scoped.
can be just:
@Service
public class RequestData {
    @Autowired
    HttpServletRequest httpServletRequest;

    public String authorsation() {
        return httpServletRequest.getHeader("authorization");
    }
}

Reply:
Thanks, changing!